.github/workflows/pythonpackage.yml

```diff
@@ -10,13 +10,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        PYTHON_VERSION: [3.6.10, 3.7.6, 3.8.2]
+        PYTHON_VERSION: [3.6.10, 3.7.7, 3.8.2]
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{matrix.PYTHON_VERSION}}
       uses: actions/setup-python@v1
       with:
         python-version: ${{matrix.PYTHON_VERSION}}
+    - name: Install bwa
```

Comment:
Isn't the user of this library going to need bwa anyways? Or is the expectation that he should install it by hand?

Reply:
The user needs to have bwa to run it.  This is for the integration tests run on github.com